### PR TITLE
Backup metal3 images

### PR DESCRIPTION
**Pull request by Sunnatillo**
This PR adds workflow to backup metal3 images in Kna1 region.

### CODE REVIEW

File: ci/images/gen_metal3_centos_image.sh [hunk 1]
```diff
@@ -88,9 +88,13 @@ packer build \
   -var "flavor=${IMAGE_FLAVOR}" \
   "${IMAGES_DIR}/${BUILDER_CONFIG_FILE}"
 
-# Replace any old image
-
-replace_image "${IMAGE_NAME}" "${FINAL_IMAGE_NAME}"
+# Replace image. Backup image if it is building Metal3 image in Kna region
+# We keep metal3 backup images in only Kna region
+if [[ "${PROVISIONING_SCRIPT}" == *"metal3-img"* && "${OS_REGION_NAME}" == "Kna1" ]]; then
```
Comment (tuminoid) [inline ci/images/gen_metal3_centos_image.sh:93]:
```suggestion
if [[ "${PROVISIONING_SCRIPT}" == *"metal3-img"* ]] && [[ "${OS_REGION_NAME}" == "Kna1" ]]; then
```
No `&&` inside `[[ .. ]]`

File: ci/images/gen_metal3_ubuntu_image.sh [hunk 2]
```diff
@@ -82,9 +82,13 @@ packer build \
   -var "flavor=${IMAGE_FLAVOR}" \
   "${IMAGES_DIR}/${BUILDER_CONFIG_FILE}"
 
-# Replace any old image
-
-replace_image "${IMAGE_NAME}" "${FINAL_IMAGE_NAME}"
+# Replace image. Backup image if it is building Metal3 image in Kna region
+# We keep metal3 backup images in only Kna region
+if [[ "${PROVISIONING_SCRIPT}" == *"metal3-img"* && "${OS_REGION_NAME}" == "Kna1" ]]; then
```
Comment (tuminoid) [inline ci/images/gen_metal3_ubuntu_image.sh:87]: same

**Review by tuminoid — COMMENTED** [review]
Nit

File: ci/images/gen_metal3_ubuntu_image.sh (hunk 2 above)
Comment (Sunnatillo) [reply ci/images/gen_metal3_ubuntu_image.sh]: Thanks. fixed

File: ci/images/gen_metal3_centos_image.sh (hunk 1 above)
Comment (Sunnatillo) [reply ci/images/gen_metal3_centos_image.sh]: fixed

**Review by tuminoid — APPROVED** [review]
/lgtm

**Review by lentzi90 — APPROVED** [review]
/lgtm

**Pull request merged**